engine-tests/src/tests/repro.rs

```diff
@@ -88,7 +88,7 @@ fn repro_5bEgfRQ() {
         block_timestamp: 1_651_073_772_931_594_646,
         input_path: "src/tests/res/input_5bEgfRQ.hex",
         evm_gas_used: 6_414_105,
-        near_gas_used: 508,
+        near_gas_used: 649,
```

Comment:
Why has the gas usage gone up so significantly on this test?

Reply:
It's expected behavior. As we're using on Engine version related to Prague. And that particular release is Cancun.
@aleksuss asked me to use previous version, for precompilles release.
But after that I'll update to new version of `aurora-evm`. And I'm expected, that all optimized gas cost for engine will return back